metsrw/utils.py

```diff
@@ -20,6 +23,17 @@ def lxmlns(arg):
 
 
 # CONSTANTS
-
```

Comment:
You're right that "block comments" didn't explain the idea. I was referring to a common pattern in Python projects where a comment surrounded by empty lines is used not just to refer to the immediate line of code below but to a whole, e.g. see [this example in the transaction module of the Django project](https://github.com/django/django/blob/495abe00951ceb9787d7f36590f71aa14c973d3d/django/db/transaction.py#L101-L106):

```
def on_commit(func, using=None):
    #...


#################################
# Decorators / context managers #
#################################

class Atomic(ContextDecorator):
    # ...


def atomic(using=None, savepoint=True):
    # ...
```

Before your change, `LXML HELPERS` was doing the same thing. I'll stop here, feel free to ignore me. My thinking here aligns with that idea Justin mentioned a couple of times: [Chesterton's fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence):

> If you're considering nominating something for deletion, or changing a policy, because it doesn't appear to have any use or purpose, research its history first. You may find out why it was created, and perhaps understand that it still serves a purpose. If you believe the issue it addressed is no longer valid, frame your argument for deletion in a way that acknowledges that.

Thanks.

Reply:
@sevein thank. I can see what you want to achieve. I'm not sure the block-comment as we had it was the right way to do that, so maybe we can adopt the Django styling.